.github/workflows/security.yml

```
@@ -0,0 +1,44 @@
+        uses: anchore/scan-action@v3
+        with:
+          image: tolgee:ci    # Should match the tag from previous step, note the colon!
+          fail-build: true
+          output-format: sarif
+          acs-report-enable: true
+
+      # Upload SARIF to show alerts in PR
+      - name: Upload SARIF
+        uses: github/codeql-action/upload-sarif@v2
+        with:
+          sarif_file: grype.sarif
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)